gotechnica/platform-2020
# Adds Backend endpoint for tracking basic user activity

Description
---

Adds the `track_user_activity` to the Users service in the backend. This endpoint adds entries to the `platform-<stage>-activity` table. In order to use it, make a post request to the endpoint with a `user_id` and `action` parameter. The `user_id`corresponds to the user that is participating in whatever activity is recorded in the `action` parameter. The entry in the table will also have a `timestamp` and `id` parameter generated within this endpoint.

Steps to Test
---

You can call the endpoint directly and verify the entry is present in the corresponding table. Also, the added `track_user_activity` test will test basic functionality of this endpoint.
